packages/network-navigator/src/NetworkNavigator.spec.ts

```diff
@@ -485,18 +485,15 @@ describe('NetworkNavigator', () => {
 		return { instance, element, singleNode }
 	}
 
-	it('should deselect the node if the same node is selected twice', () => {
+	it('should deselect the node if the same node is selected twice', (done: DoneFn) => {
 		const { instance, singleNode } = selectTest()
-
-		// Same node clicked twice
-		performClick(singleNode[0])
-
-		// Everything should be deselected since we toggled the same node
-		const selected = instance.data.nodes.find(d => d.selected)
-		expect(selected).to.equal(
-			undefined,
-			'should deselect the node if the same node is selected twice',
-		)
+		setTimeout(() => {
+			// Everything should be deselected since we toggled the same node
+			performClick(singleNode[0])
+			const selected = instance.data.nodes.find(d => d.selected)
+			expect(selected).to.equal(undefined, 'no node should be selected')
+			done()
+		}, 500)
```

Comment:
I'm a little worried about the 500 here; it seems like this could lead to a flaky test. Does a zero work just as well by putting this in the Queue?

Reply:
I would guess that if any waiting is appropriate, it's after we emit the click event so that the system can respond. I'm not sure selectTest() is doing anything worth waiting around for

Reply:
You're right. I was having some problems with this test here, but it wasn't related to timing as I imagined. I fixed this test to a different approach, and we do not need the timeout anymore